packages/web/src/common/store/tipping/sagas.ts

```diff
@@ -264,11 +264,19 @@ function* confirmTipIndexed({
 
 function* wormholeAudioIfNecessary({ amount }: { amount: number }) {
   const walletClient = yield* getContext('walletClient')
+  const sdk = yield* getSDK()
+  const { currentUser } = yield* select(getWalletAddresses)
```

Comment:
Most nittiest of nits: `currentUser` is pretty widely used in sagas to mean that actual `User`, so this variable name is a little confusing. But not blocking.

Reply:
yeah it's the name of the thing returned from the selector but ur right